### ISSUE TITLE
XML-Format broken in Windows

### ISSUE THREAD

**Issue by lowell80** [Kintyre/ksconf#52]
The `xml-format` command seems to be broken on Windows.  Or at the very least, the unit tests are broken.

Fix this before 0.7.3 release.

**Comment by lowell80** [comment]
```
ksconf --version
 _                         __
| | _____  ___ ___  _ __  / _|
| |/ / __|/ __/ _ \| '_ \| |_
|   <\__ \ (_| (_) | | | |  _|
|_|\_\___/\___\___/|_| |_|_|
ksconf 0.7.3rc3.dev2+b71f345
Python: 2.7.16  (c:\python27\python.exe)
Git SHA1 b71f3458 committed on 2019-06-04
Installed at: c:\python27\lib\site-packages\ksconf
Written by Lowell Alleman <lowell@kintyre.co>.
Copyright (c) 2019 Kintyre Solutions, Inc, all rights reserved.
Licensed under Apache Public License v2
  kintyre_splunk_conf  (0.7.3rc3.dev2+b71f345-py2.7)
    Commands:
      check           (stable)    OK
      combine         (beta)      OK
      diff            (stable)    OK
      filter          (alpha)     OK
      merge           (stable)    OK
      minimize        (beta)      OK
      promote         (beta)      OK
      rest-export     (beta)      OK
      rest-publish    (alpha)     OK   (splunk-sdk 1.6.6)
      snapshot        (alpha)     OK
      sort            (stable)    OK
      unarchive       (beta)      OK
      xml-format      (alpha)     OK   (lxml 4.3.3)
coverage run -m unittest discover -s tests
..............ss........................................................Exception while running: ksconf xml-format c:\users\appveyor\appdata\local\temp\1\tmpahgex3-ksconftest\sample.xml
STDOUT:
Completed formatting 1 files.  rc=100 Breakdown:
   0 files were formatted successfully.
   0 files were already formatted.
   1 files failed.
STDERR:
Unhandled top-level exception while parsing c:\users\appveyor\appdata\local\temp\1\tmpahgex3-ksconftest\sample.xml.  Aborting.
[Errno 22] invalid mode ('r') or filename: u'file:/c:/users/appveyor/appdata/local/temp/1/tmpahgex3-ksconftest/sample.xml'
F...Exception while running: ksconf xml-format c:\users\appveyor\appdata\local\temp\1\tmpujzf8_-ksconftest\sample.xml
STDOUT:
Completed formatting 1 files.  rc=100 Breakdown:
   0 files were formatted successfully.
   0 files were already formatted.
   1 files failed.
STDERR:
Unhandled top-level exception while parsing c:\users\appveyor\appdata\local\temp\1\tmpujzf8_-ksconftest\sample.xml.  Aborting.
[Errno 22] invalid mode ('r') or filename: u'file:/c:/users/appveyor/appdata/local/temp/1/tmpujzf8_-ksconftest/sample.xml'
FException while running: ksconf xml-format c:\users\appveyor\appdata\local\temp\1\tmplo7n9s-ksconftest\sample.xml
STDOUT:
Completed formatting 1 files.  rc=100 Breakdown:
   0 files were formatted successfully.
   0 files were already formatted.
   1 files failed.
STDERR:
Unhandled top-level exception while parsing c:\users\appveyor\appdata\local\temp\1\tmplo7n9s-ksconftest\sample.xml.  Aborting.
[Errno 22] invalid mode ('r') or filename: u'file:/c:/users/appveyor/appdata/local/temp/1/tmplo7n9s-ksconftest/sample.xml'
FException while running: ksconf xml-format c:\users\appveyor\appdata\local\temp\1\tmph4weyg-ksconftest\sample.xml
STDOUT:
Completed formatting 1 files.  rc=100 Breakdown:
   0 files were formatted successfully.
   0 files were already formatted.
   1 files failed.
STDERR:
Unhandled top-level exception while parsing c:\users\appveyor\appdata\local\temp\1\tmph4weyg-ksconftest\sample.xml.  Aborting.
[Errno 22] invalid mode ('r') or filename: u'file:/c:/users/appveyor/appdata/local/temp/1/tmph4weyg-ksconftest/sample.xml'
F...............................
======================================================================
FAIL: test_already_sorted (test_cli_xmlformat.CliXmlFormatTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\ksconf\tests\test_cli_xmlformat.py", line 155, in test_already_sorted
    self.assertEqual(ko.returncode, EXIT_CODE_SUCCESS)
AssertionError: 100 != 0
======================================================================
FAIL: test_no_indent (test_cli_xmlformat.CliXmlFormatTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\ksconf\tests\test_cli_xmlformat.py", line 119, in test_no_indent
    self.assertEqual(ko.returncode, EXIT_CODE_FORMAT_APPLIED)
AssertionError: 100 != 8
======================================================================
FAIL: test_preserve_CDATA (test_cli_xmlformat.CliXmlFormatTest)
<![CDATA[]]> blocks should be preserved, even if there's no special characters.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\ksconf\tests\test_cli_xmlformat.py", line 181, in test_preserve_CDATA
    self.assertEqual(ko.returncode, EXIT_CODE_FORMAT_APPLIED)
AssertionError: 100 != 8
======================================================================
FAIL: test_promote_to_CDATA (test_cli_xmlformat.CliXmlFormatTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\ksconf\tests\test_cli_xmlformat.py", line 172, in test_promote_to_CDATA
    self.assertEqual(ko.returncode, EXIT_CODE_FORMAT_APPLIED)
AssertionError: 100 != 8
----------------------------------------------------------------------
Ran 110 tests in 3.044s
FAILED (failures=4, skipped=2)
Command exited with code 1
```

**Comment by lowell80** [comment]
Fixed in v0.7.3
